docs/server-setup.md

```diff
@@ -33,11 +33,11 @@ Github link:
 
 1. Download the files above.
 
-1. Create a new folder within the build folder called **Data**. i.e **E:\Build\AzerothCore\RelWithDebInfo\Data**
+1. Create a new folder within the root of your repo called **Data**. i.e **E:\azerothcore-wotlk\data\ (Windows)** or **/home/$USER/azerothcore/data (Linux/macOS)** (if you want to keep multiple sets of data you may want to create further levels below this, but if this is not explicitly what you need just leave it all flat under your newly created data folder).
```

Comment:
this is just wrong

Reply:
```suggestion
1. Create a new folder within the build directory called **Data**. i.e **$\Build\RelWithDebInfo\Data**
```

Reply:
Why I say it is wrong is because the config option default points to "." which is the same directory where the worldserver is. So \build\RelWithDebInfo\